LeagueInformer/LeagueInformer/Services/GetSummonerGameService.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Threading.Tasks;
+using LeagueInformer.Api;
+using LeagueInformer.Enums;
+using LeagueInformer.Interfaces;
+using LeagueInformer.Models;
+using Newtonsoft.Json.Linq;
+
+namespace LeagueInformer.Services
+{
+    public class GetSummonerGame : IGetSummonerGame
+    {
+        private readonly ApiClient _apiClient = new ApiClient();
+
+        public async Task<SummonerGame> GetSummonerGameInformation(string id)
+        {
+            try
+            {
+
+                JObject response = JObject.Parse(await _apiClient.GetJsonFromUrl(
+                    $"https://eun1.api.riotgames.com/lol/spectator/v4/active-games/by-summoner/{id}?api_key={AppSettings.AuthorizationApiKey}"));
+
```

Comment:
Sformatuj to inaczej np: 
`             return response == null
                    ? new SummonerGame { IsSuccess = false } 
                    : new SummonerGame
                    {
                        IsSuccess = true,
                        gameMode = response.GetValue("gameMode").ToString()
                    };`

Reply:
Tak żeby ? był w nowej linijce ze swoim warunkiem i : ze swoim warunkiem